### PR TITLE
[ci] Fix Gradle cache in Android workflows

### PR DESCRIPTION
**Pull request by ntadej**
It seems the expressions were not done properly, probably due to some find-replace.

Also remove trailing whitespace.

### CODE REVIEW

File: .github/workflows/android-release.yml [hunk 1]
```diff
@@ -14,21 +14,27 @@ jobs:
...
-
       - uses: actions/checkout@v4
         with:
           submodules: recursive
           fetch-depth: 0
 
+      - name: Setup cmake
+        uses: jwlawson/actions-setup-cmake@v1.14
+        with:
+          cmake-version: "3.19.x"
```
Comment (louwers) [inline .github/workflows/android-release.yml:25]: CMake 3.27.6 is included by default.

https://github.com/actions/runner-images/blob/main/images/linux/Ubuntu2204-Readme.md#tools

Comment (ntadej) [reply .github/workflows/android-release.yml:25]: This just unifies it with the CI. I can revert it from both.

Comment (louwers) [reply .github/workflows/android-release.yml:25]: Ah! Yes please.

Comment (ntadej) [reply .github/workflows/android-release.yml]: Done.

**Review by louwers — APPROVED** [review]

**Pull request merged**